src/Assimp/Silk.NET.Assimp.Tests/GlobalUsings.cs

```diff
@@ -0,0 +1 @@
+global using Xunit;
```

Comment:
Just noting that if you would like to avoid having files such as this one, you can put this in the project file:

```xml
    <ItemGroup>
        <Using Include="Xunit" />
    </ItemGroup>
```

Reply:
Good point, Rider generated this as part of the template. Will fix.